dart/lib/src/scope.dart

```diff
@@ -190,22 +184,23 @@ class Scope {
       }
     });
 
+    SentryEvent? processedEvent = event;
```

Comment:
why are we copying instead of using the same instance? we are mutating it anyway

Reply:
Because the passed instance is non-nullable but because of the processing it can now be nullable. Thus we need to declare a new variable which is nullable.

Reply:
makes sense, missed that, thanks